manifests/Docker/DockerDesktopEdge/2.3.1.0.45408.yaml

```diff
@@ -0,0 +1,25 @@
+# ================== General ==================
+
+Id: Docker.DockerDesktopEdge
+Publisher: Docker Inc.
+Name: Docker Desktop Edge
+AppMoniker: docker
+Description: Docker Desktop is an application for MacOS and Windows machines for the building and sharing of containerized applications. Access Docker Desktop and follow the guided onboarding to build your first containerized application in minutes.
+Homepage: https://www.docker.com
+Tags: Container, Containerization, Virtualization
+
+Version: 2.3.1.0.45408
```

Comment:
Why 5 numbers?
According to https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv0.1.md there should be only 4, the installer version also does not contain our internal build number. It's part of the download URL, but shouldn't appear here as a `winget search` then looks more complicated

```
PS C:\Users\stefan\Desktop> winget search docker
Name                Id                       Version       Matched
--------------------------------------------------------------------------
Docker Desktop      Docker.DockerDesktop     2.3.0.3.45519 Moniker: docker
Docker Desktop Edge Docker.DockerDesktopEdge 2.3.1.0.45408 Moniker: docker
```

Reply:
> does not contain our internal build number. It's part of the download URL

You sure? https://download.docker.com/win/stable/appcast.xml

> According to https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv0.1.md there should be only 4

As of now, you can ignore the specification.

Reply:
Alright, so let's keep it as it is right now. Thanks for the details.